crates/astria-cli/src/types.rs

```diff
@@ -73,11 +73,11 @@ impl RollupDeploymentConfig {
     }
 
     #[must_use]
-    pub fn get_initial_sequencer_height(&self) -> u64 {
-        self.sequencer.initial_block_height
+    pub fn get_initial_sequencer_height(&self) -> String {
```

Comment:
Could to `-> &str`

Reply:
This getter/setter pair is never used. Going to remove